depin/services/roles/xai/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+xai_cmd: "{{ depin_cmd | default('install') }}"
+xai_version: 1.1.14
+xai_metrics_port: 33004
+xai_download_url: "https://github.com/xai-foundation/sentry/releases/latest/download/sentry-node-cli-linux.zip"
+
+xai_install_path: "/opt/xai/{{ xai_version }}"
+
+xai_node_count: 1
+
+xai_node_port: 8545
+
+xai_start_command: "/opt/xai/{{ xai_version }}/sentry-node-cli-linux start"
+
+xai_stop_command: "/opt/xai/{{ xai_version }}/sentry-node-cli-linux stop"
```

Comment:
Should these be here or moved to `tasks/commands/start` and `tasks/commands/stop` respectively?

Reply:
removed